packages/api/src/providers/fitbit.ts

```diff
@@ -470,22 +465,36 @@ export class Fitbit extends Provider implements OAuth2 {
    * @returns List of active FitbitWebhookSubscriptions or undefined
    */
   async getActiveSubscriptions(
-    accessToken: string
-  ): Promise<FitbitWebhookSubscriptions | undefined> {
-    const url = `${Fitbit.URL}/${Fitbit.API_PATH}/apiSubscriptions.json`;
-    try {
-      const resp = await axios.get(url, {
-        headers: {
-          Authorization: `Bearer ${accessToken}`,
-        },
-      });
-      if (resp.status === status.OK)
-        return fitbitWebhookSubscriptionSchema.parse(resp.data.apiSubscriptions);
-    } catch (err) {
-      console.log(`Failed to fetch active subscriptions. Url: ${url}, Error: ${err}.`);
-      capture.error(err, { extra: { context: `fitbit.getActiveSubscriptions`, url, err } });
-      throw new Error("Failed to get active WH subscriptions for Fitbit", { cause: err });
-    }
+    accessToken: string,
+    scopes: string
```

Comment:
Update the TSDoc w/ new params.

The description of the return is misleading: we're saying it returns a list of `FitbitWebhookSubscriptions`, but that type is the list itself. It would be better to have `FitbitWebhookSubscription` as a single subscription and return `FitbitWebhookSubscription[]` instead. But NABD if we don't want to change this on this PR, let's at least fix the description, please - and ticket the fix or add it to another ongoing PR.

Also, is scopes a comma-separated string? Not an array of strings?

Reply:
Updated the FitbitWebhookSubscription to be a single subscription object. 
Scopes is just a space-separated string, i.e. `sleep weight profile settings` etc.